src/gen_modules_config.F90

```diff
@@ -9,86 +9,116 @@ module g_config
   use o_param
   implicit none
   save
-
+  !_____________________________________________________________________________
   ! *** Modelname ***
   character(5)           :: runid='test1'       ! a model/setup name
   namelist /modelname/ runid
-
+  
+  !_____________________________________________________________________________
   ! *** time step ***
   integer                :: step_per_day=72     ! number of steps per day
   integer                :: run_length=1	! run length
   character              :: run_length_unit='y' ! unit: y, d, s
   namelist /timestep/ step_per_day, run_length, run_length_unit
   
+  !_____________________________________________________________________________
   ! *** Paths for all in and out ***
   character(100)         :: MeshPath='./mesh/'
   character(100)         :: ClimateDataPath='./hydrography/'
   character(100)         :: TideForcingPath='./tide_forcing/'
   character(1000)        :: ResultPath='./result/'
   namelist /paths/  MeshPath, ClimateDataPath, &
        TideForcingPath, ResultPath
-
+       
+  !_____________________________________________________________________________
   ! *** restart_log ***
   integer                :: logfile_outfreq=1      ! logfile info. outp. freq., # steps
   integer                :: restart_length=1
   character              :: restart_length_unit='m'
   
   namelist /restart_log/   restart_length, restart_length_unit, logfile_outfreq
 
+  !_____________________________________________________________________________
   ! *** ale_def ***
-  character(20)          :: which_ALE='linfs' ! 'zlevel', 'zstar', 'zstar-weighted', 'ztilde'
-  logical                :: use_partial_cell=.false.  ! use partial bottom cell configuration  
+  ! Which ALE case to use : 'linfs', 'zlevel', 'zstar'
+  character(20)          :: which_ALE='linfs' 
+  
+  ! use partial bottom cell configuration .true./.false. 
+  logical                :: use_partial_cell=.false.  
+  
+  ! if a thicker partial bottom layer thickness is more realistic than always 
+  ! apply it, BUT when a thinner partial bottom layer thickness is more realistic than 
+  ! only apply it when the initial full cell bottom layer thickness is above the 
+  ! treshhold partial_cell_tresh to not allow already thin layers to become even 
+  ! thinner. e.g 
+  ! partial_cell_tresh=10 --> thinner partial bottom cells will be only applied for initial 
+  ! bottom layer thicknesses larger than 10m 
+  real(kind=WP)          :: partial_cell_tresh=0.0_WP 
```

Comment:
Maybe rename to `partial_cell_thresh` :)

Reply:
How about partial_cell_trash ?! ;-D

Reply:
That's how I read it initially :))